scripts/zones/AlTaieu/mobs/Jailer_of_Prudence.lua

```diff
@@ -19,6 +24,17 @@ end;
 -----------------------------------
 
 function onMobSpawn(mob)
+    mob:AnimationSub(0); -- Mouth closed
+    mob:addStatusEffect(EFFECT_FLEE,100,0,30);
```

Comment:
Use addStatusEffectEx instead of addStatusEffect. Same parameters, just a different function name.